apps/yapms/src/lib/components/modals/editregionmodal/EditRegionModal.svelte

```diff
@@ -4,6 +4,7 @@
 	import ModalTitle from '../../modalutilities/ModalTitle.svelte';
 
 	$: open = $EditRegionModalStore.open;
+	$: confirmDisabled = $EditRegionModalStore.confirmDisabled;
 	$: longName = open ? $EditRegionModalStore.region?.longName : undefined;
 	$: value = open ? $EditRegionModalStore.region?.permaVal : undefined;
 	$: newValue = value ?? 0;
```

Comment:
Maybe have a dynamic "newValueOutOfBounds" variable here that does the logic for checking outOfBounds.

Reply:
```js
$: newValueOutOfBounds = newValue < 0;
```